.gitignore

```diff
@@ -1,20 +1,20 @@
-/build/
-!/build/cache/.gitkeep
+!/.tools/.gitkeep
 !/build/.gitkeep
+!/build/cache/.gitkeep
```

Comment:
Not a big fan of a single project-wide .gitignore tbh.

Reply:
By experience with Symfony, it is way more confusing for people in general to have gitignore files left and right.

Reply:
True, having `.gitignore` left and right isn't right too, pun intended.

Reply:
Anyway, it is already as it is, and this is not about it here.